tests/rt.conf

```diff
@@ -165,21 +165,21 @@ RUN     | cpld_restart_bmarkfrac
 RUN     | cpld_bmarkfrac_v16                                                                                                      | - wcoss_cray gaea.intel jet.intel       | fv3 |
 RUN     | cpld_restart_bmarkfrac_v16                                                                                              | - wcoss_cray gaea.intel jet.intel       |     | cpld_bmarkfrac_v16
 
-COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled S2S=Y WW3=Y         | - wcoss_cray gaea.intel jet.intel       | fv3 |
+COMPILE | APP=S2SW SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled            | - wcoss_cray gaea.intel jet.intel       | fv3 |
 RUN     | cpld_bmark_wave                                                                                                         | - wcoss_cray gaea.intel jet.intel       | fv3 |
 RUN     | cpld_bmarkfrac_wave                                                                                                     | - wcoss_cray gaea.intel jet.intel       | fv3 |
 RUN     | cpld_bmarkfrac_wave_v16                                                                                                 | - wcoss_cray gaea.intel jet.intel       | fv3 |
 RUN     | cpld_control_wave                                                                                                       | - wcoss_cray gaea.intel jet.intel       | fv3 |
 
-COMPILE | DEBUG=Y SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled S2S=Y       | - wcoss_cray jet.intel                  | fv3 |
+COMPILE | APP=S2S DEBUG=Y SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled     | - wcoss_cray jet.intel                  | fv3 |
 RUN     | cpld_debug                                                                                                              | - wcoss_cray jet.intel                  | fv3 |
 RUN     | cpld_debugfrac                                                                                                          | - wcoss_cray jet.intel                  | fv3 |
 
 ###################################################################################################################################################################################
 # Data Atmosphere tests                                                                                                                                                           #
 ###################################################################################################################################################################################
 
-COMPILE | DATM=Y S2S=Y                                                                                                            | - wcoss_cray jet.intel                  | fv3 |
+COMPILE | APP=DATM_NEMS
```

Comment:
Add` | - wcoss_cray jet.intel                  | fv3 |`

Reply:
addressed in 277cefb